### PR TITLE
CE-18929 : Updating docker binary version to 1.9.9

### PR DESCRIPTION
**Pull request by Ramesh7**
##JIRA

[JIRA](https://coupadev.atlassian.net/browse/CE-18929)

## Summary

- Updating vault binary version to update the [go](https://github.com/hashicorp/vault/tree/v1.9.9) library version

### CODE REVIEW

**Comment by coupaopsin** [comment]
:rotating_light: CI Pipeline is in FAILED state - [[#Build-31](https://jenkins-opstest.coupadev.com/job/Cloud_Engineering/job/docker-vault-ci-pipeline/31/)]

**Comment by Ramesh7** [comment]
rerun tests

**Comment by coupaopsin** [comment]
:white_check_mark: CI Pipeline completed successfully - [[#Build-37](https://jenkins-opstest.coupadev.com/job/Cloud_Engineering/job/docker-vault-ci-pipeline/37/)]

File: 0.X/Dockerfile [hunk 1]
```diff
@@ -1,7 +1,7 @@
 FROM alpine:3.16
 
 # This is the release of Vault to pull in.
-ARG VAULT_VERSION=1.9.8
+ARG VAULT_VERSION=1.9.9
```
Comment (pkazi) [inline 0.X/Dockerfile:4]: just for history
as per twistlock scan report - 
go version 1.17.13 has 1 vulnerability
which is Fixed in: 1.19.1, 1.18.6
https://twistlock.coupahost.com:8083/#!/monitor/vulnerabilities/images/ci?search=sha256:c99567af21236372d9f7caefa215059a8e40d84d55b97b2a17adb1978774c5a6

but Vault does not have any stable release with these go versions.

**Review by pkazi — APPROVED** [review]
lgtm

**Review by rahuls2997 — APPROVED** [review]

**Comment by coupaopsin** [comment]
:rotating_light: CI Pipeline is in FAILED state - [[#Build-42](https://jenkins-opstest.coupadev.com/job/Cloud_Engineering/job/docker-vault-ci-pipeline/42/)]

**Pull request merged**